go/pkg/tree/tree.go

```diff
@@ -66,7 +66,11 @@ func loadFiles(execRoot string, excl []*command.InputExclusion, path string, fs
 	meta := cache.Get(absPath)
 	t := command.FileInputType
 	if meta.Err != nil {
-		if e, ok := meta.Err.(*filemetadata.FileError); !ok || !e.IsDirectory {
+		e, ok := meta.Err.(*filemetadata.FileError)
```

Comment:
if !ok {
  return meta.Err
}
and from there on ignore ok

Reply:
Done